pkg/bootstrapper/bootstrapper.go

```diff
@@ -253,6 +263,8 @@ type fileTranslation struct {
 type kubeletConf struct {
 	// ClientCAFile specifies location to client certificate
 	ClientCAFile string
+	// clusterDNSIP is the IP address used to configure the kubelet with a specific clusterDNS IP
```

Comment:
Considering this struct describes the kubelet configuration I feel you can drop the `configure the kubelet` part, and instead go with something like:
`clusterDNSIP is the IP address of the DNS server used for all containers`